apps/synapse-portal-framework/package.json

```diff
@@ -33,26 +31,12 @@
   "scripts": {
     "clean": "rimraf coverage",
     "lint": "eslint src",
-    "copy-assets": "cp src/config/style/favicon* public/; cp src/config/style/logo.svg public/; cp src/config/style/socialmedia.png public/",
```

Comment:
These files are now just put in the `public` folder of the individual portal project, which makes this `cp` step unnecessary. So you will not see it in the new portal projects.